index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <!-- Markdown Preview project: R Pulcer -->
+  <title>Markdown Preview</title>
+
+  <style>
+    #source {
+      float: left;
+      width: 45%;
+      height: 300px;
+      padding: 2%;
+    }
+    #preview {
+      float: right;
+      width: 45%;
+      height: 300px;
+      padding: 2%;
+      border: solid;
+      border-width: 1px;
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+      font-size: 14px;
+    }
+    button {
+      padding: 5px;
+      margin: 5px;
+    }
+</style>
+
+</head>
+
+<body>
+<h1>Markdown Preview</h1>
+
+<!--
+<button id="render" onclick="process()">Render</button>
+-->
+<button id="render">Render</button>
+<button id="bold">Bold</button>
+<button id="italics">Italics</button>
+<button id="clear">Clear</button>
+<button id="tester">TEST</button>
+<br /><br />
+
+<textarea id="source" onkeyup="process()" placeholder="Enter text with markdown short codes here...">
+</textarea>
+
+<div id="preview">
+</div>
+
+
+<script src='./node_modules/marked/lib/marked.js'></script>
+
+<script>
+  const src = document.getElementById("source");
```

Comment:
Personally I prefer variable names to be spelled out -- e.g. `source` instead of `src` here. In VS Code, F2 lets you quickly rename a symbol.